### ISSUE TITLE
Can't iterate over connections

### ISSUE THREAD

**Issue by pawel-ch** [marcsello/pyopenvidu#3]
* PyOpenVidu version: 0.1.1
* Python version: 3.8
* Operating System: Ubuntu 18.04, 19.10
* `OpenVidu.get_config()`

```python
{
    'version': '2.12.0',
    'kmsUris': ['ws://localhost:8888/kurento'],
    'openviduPublicurl': 'https://<our-test-domain>:4443/',
    'openviduCdr': False,
    'maxRecvBandwidth': 1000,
    'minRecvBandwidth': 300,
    'maxSendBandwidth': 1000,
    'minSendBandwidth': 300,
    'openviduRecording': False,
    'openviduWebhook': False
}
```

### Description

When I wanted to iterate over session's connections I've ran into problem with https://github.com/marcsello/pyopenvidu/blob/0.1.1/pyopenvidu/openvidusubscriber.py#L20 - there's no `publisher` in any of `subscribers`. The only fields available I can see in debugger are `createdAt` and `streamId`.

### What I Did

**Step 1**

```
import pyopenvidu
import uuid

client = pyopenvidu.OpenVidu("<openvidu-url>", "<openvidu-secret>")
custom_session_id = str(uuid.uuid4())
try:
    session = self.client.get_session(custom_session_id)
except pyopenvidu.exceptions.OpenViduSessionDoesNotExistsError:
    session = self.client.create_session(custom_session_id)
token1 = session.generate_token(data="<user1-email>")
token2 = session.generate_token(data="<user2-email>")
```

**Step 2**

After that I've connected to OpenVidu via websocket once using `token1` and twice using `token2` resulting in 3 active connections. The goal is to identify someone using `token2` twice and invalidating his older connection.

**Step 3**

```
client = pyopenvidu.OpenVidu("<openvidu-url>", "<openvidu-secret>")
client.fetch()
for session in client.sessions:
    if session.connection_count != 3:
        continue
    for connection in session.connections:
        # do anything with connection
```

**Comment by marcsello** [comment]
I was able to reproduce the issue. It seems like that this problem is caused by that the documentation I've based my implementation on is inconsistent with the actual API provided by the server.

In the official API documentation by OpenVidu, it is stated, that a subscriber object contains a publisher field:
https://docs.openvidu.io/en/2.12.0/reference-docs/REST-API/#get-apisessionsltsession_idgt

But it seems like in practice it does not. 
I'm not sure if this field is optional, or just no longer exists in OpenVidu. I'll try to dig further.

I'm considering setting a default `None` value to missing fields. I'm not sure how would this affect the ease of use, as developers have to handle those None values.

**Comment by marcsello** [comment]
As it is stated in the referenced issue at OpenVidu... This field is removed. So I am going to remove that field as well.

**Comment by marcsello** [comment]
This issue is fixed in release 0.1.2.
Thank you for helping!

**Comment by pawel-ch** [comment]
Thank you for quick response and fix :)
